src/server/actions.ts

```diff
@@ -27,3 +27,20 @@ export async function fetchCountries(searchTerm: string): Promise<any> {
     return { error: error };
   }
 }
+
+export async function searchCountryByCode(code: string): Promise<any> {
```

Comment:
Hi @jakemelvin. The use of any is forbidden in Typescript projects. If not, why are we using typescript? Your method doesn't just return any Promise. It returns a Promise of a country. So what you need to do, is to define an interface for a country response, e.g

`export interface GetCountryResponseData {
id : string
name: string
//etc
}`

and your Promise will be a Promise of a Country like this:

Promise<GetCountryResponseData>